src/app/components/StakingDateSelector/index.tsx

```diff
@@ -193,8 +167,26 @@ export function StakingDateSelector(props: Props) {
           );
         })}
       </div>
-      <div className="sliderMonth tw-mt-5 tw-pr-0">
-        <Slider {...settingsSliderMonth}>
+      <div className="sliderMonth tw-mt-5 tw-pr-0 tw-relative">
+        <Carousel
+          arrows={false}
+          partialVisible={false}
+          responsive={{
+            desktop: {
+              breakpoint: { max: 4800, min: 0 },
+              items: 6,
+              slidesToSlide: 6,
+            },
+          }}
+          draggable
+          focusOnSelect={false}
+          minimumTouchDrag={80}
+          customDot={<CustomDot />}
+          showDots
+          customButtonGroup={<CustomButtonGroup />}
+          renderButtonGroupOutside
+          swipeable
+        >
```

Comment:
When switching to the last year, the buttons and the dots lag 1-2 seconds behind, could you please check if this can be avoided, maybe by passing the entry count to the carousel directly?

Reply:
I tried multiple configurations but nothing helped with the issue :-/ . However, I adjusted it and in case there is no need for a slider, I don't show arrows anymore.

Reply:
See new change request.